torch/_decomp/decompositions.py

```diff
@@ -3980,6 +3980,28 @@ def scaled_dot_product_flash_attention(
     )
 
 
+@register_decomposition(aten.full_like)
+def full_like(
+    self,
+    fill_value,
+    *,
+    dtype=None,
+    layout=None,
+    device=None,
+    pin_memory=False,
+    requires_grad=False,
+    memory_format=torch.preserve_format,
+):
+    return torch.full(
+        [*self.size()],
```

Comment:
This is wrong because `torch.full_like` should preserve the stride order of `self` in the same way that `empty_like` does. This decomposition works fine for inductor though because we don't care about strides.

Reply:
@peterbell10 So what can I do to preserve stride order? I tried using (...).as_strided(...) but it did not help